uber/site_sections/accounts.py

```diff
@@ -16,10 +16,11 @@ def index(self, session, message=''):
             'message':  message,
             'accounts': session.query(AdminAccount).join(Attendee)
                                .order_by(Attendee.first_name, Attendee.last_name).all(),
```

Comment:
I also made this load significantly faster than before by not bothering to parse into an ``Attendee`` object for each row.